### ISSUE TITLE
Ensure tests are fully deterministic

### ISSUE THREAD

**Issue by mmore500** [mmore500/hstrat#51]
```
 E           assert 0.9996707056754717 < 0.999
  E            +  where 0.9996707056754717 = <bound method rv_discrete.cdf of <scipy.stats._discrete_distns.binom_gen object at 0x7f5033bcf160>>(n=20, p=(1 - 0.95), k=5)
  E            +    where <bound method rv_discrete.cdf of <scipy.stats._discrete_distns.binom_gen object at 0x7f5033bcf160>> = <scipy.stats._discrete_distns.binom_gen object at 0x7f5033bcf160>.cdf
  E            +      where <scipy.stats._discrete_distns.binom_gen object at 0x7f5033bcf160> = stats.binom
  
  test_hstrat/test_phylogenetic_inference/test_pairwise/test_calc_ranks_since_mrca_bounds_with.py:462: AssertionError
```

**Comment by JJ** [comment]
No need to do that. Using a common seed will only cause trouble down the line. Even with a single and common seed, additional generations will throw everything off-kilter.
In these cases, as usual, what you need to do is to ensure the *probabilistic*  behavior holds.

**Comment by mmore500** [comment]
Closed by #58.
